src/core.js

```diff
@@ -3,21 +3,34 @@ const teachers = ['Nathan', 'Ed', 'Steve', 'Phil', 'Carlo', 'Lewis', 'Shahzad']
 
 // 1. Using an index on the teachers array, change the value of the
 // fourthTeacher variable below to be the fourth teacher listed in the array
-const fourthTeacher = undefined
+const fourthTeachers = teachers[3];
+console.log(fourthTeachers);
 
 // 2. Replace the fifth teacher in the teachers array with 'Patrick'
+teachers[4]='Patrick';
+console.log(teachers);
 
 // 3. Remove the last teacher from the array and save them in lastTeacher below
 const lastTeacher = undefined
+lastTeacher = teachers.pop();
```

Comment:
this will not work nor run because you are trying to re-update a constant variable which has an initial value of undefined. you must combine both lines of code together

`const variable = value`